### DISCUSSION TITLE
Renamed files are not detected

### DISCUSSION THREAD

**Discussion by abhatt-rh** [wjdp/htmltest#224] [Q&A]
When I run the htmltest on files that are renamed, I get a _Non-OK status: 404_ for the said files
The links works fine locally. It seems that the `htmltest.yml` does not detect the renaming. Is there a way to define this in the YAML so that the newly updated file names are considered in the test run? 
Here's my htmltest.yml 
```
DirectoryPath: public/
IgnoreDirectoryMissingTrailingSlash: true
IgnoreCanonicalBrokenLinks: false
TestFilesConcurrently: true 
```
Thanks in advance!

**Comment by abhatt-rh** [comment]
Hi @wjdp,
When you get a chance, could you PTAL? Thanks!

**Comment by wjdp** [comment]
Hey @abhatt-rh I don't think I've got enough information to help here. It sounds like there's an issue with picking up files in the `public/` directory after you rename them? htmltest doesn't cache anything about local links so unsure what the issue is.

Can you provide a worked example? List out the steps so someone else can reproduce the issue.

**Reply by abhatt-rh** [reply]
Thanks for your response.

If it helps , here's the link to the failed test:
https://github.com/validatedpatterns/docs/actions/runs/6908682855/job/18798476345#step:8:1

**Reply by wjdp** [reply]
Looking at the output it seems these are external links. For example `https://validatedpatterns.io/patterns/medical-diagnosis/med-getting-started/` which does return a 404.

```
$ http --headers https://validatedpatterns.io/patterns/medical-diagnosis/med-getting-started/
HTTP/1.1 404 Not Found
```

I think this just falls under normal ignoring of links, see https://github.com/wjdp/htmltest#see_no_evil-ignoring-content
I'm still not sure what you mean by these being 'renamed files'

**Reply by abhatt-rh** [reply]
Well to take the example of the error you cited: The [getting-started](https://github.com/validatedpatterns/docs/pull/310/files#diff-341bc602188a0d1501e5188f014f75bb85f4d6de0caa944f2f2cb420bdd4f990) file was renamed to [med-getting-started](https://github.com/validatedpatterns/docs/pull/310/files#diff-315a288e5b2e7b11b4818320afad25c621ad89bcee1fb9a9454a284f2de41f47). It builds fine locally but when the html test is run, it cannot _see_ the URL with the updated file name (https://validatedpatterns.io/patterns/medical-diagnosis/med-getting-started/) and fails with a 404 since it cannot find the older URL with the older file name (https://validatedpatterns.io/patterns/medical-diagnosis/getting-started/)

**Comment by abhatt-rh** [comment]
@wjdp Is there a way to ignore these errors considering that the renamed files is not detected and so the test fails for _false positives_
